pkg/tarsum/tarsum_test.go

```diff
@@ -251,6 +291,33 @@ var (
 	sha512Hash = NewTHash("sha512", sha512.New)
 )
 
+// Test all the build-in read size : buf8K, buf16K, buf32K and more
+func TestTarSumsReadSize(t *testing.T) {
+	// Test this on the four first layer
+	for i := 1; i < 6; i++ {
```

Comment:
for i, layer := range testLayers {
    // ...
    nbByteToRead := (i  + 1)* 8 * 1024
    // ...
}